app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsPresenter.java

```diff
@@ -29,6 +29,8 @@
     private final TicketRepository ticketRepository;
     private final DatabaseChangeListener<Ticket> ticketChangeListener;
 
+    public static boolean isNewTicketCreated = false;
```

Comment:
static variables are really bad

Reply:
I had tried injecting `TicketsPresenter` but it didn't work as needed. Then, what should be done?

Reply:
I'll see tomorrow

Reply:
@iamareebjamal Please have a look.

Reply:
@iamareebjamal Please suggest something for this.

Reply:
Skip this issue for now

Reply:
Okay